app/src/main/java/fr/free/nrw/commons/achievements/AchievementsActivity.java

```diff
@@ -247,10 +277,38 @@ private void setAchievementsUploadCount(Achievements achievements, int uploadCou
      * @param uploadCount
      */
     private void setUploadProgress(int uploadCount){
-        imagesUploadedProgressbar.setProgress
-                (100*uploadCount/levelInfo.getMaxUploadCount());
-        imagesUploadedProgressbar.setProgressTextFormatPattern
-                (uploadCount +"/" + levelInfo.getMaxUploadCount() );
+        if (uploadCount==0){
+            setZeroAchievements();
+        }else {
+
+            imagesUploadedProgressbar.setProgress
+                    (100*uploadCount/levelInfo.getMaxUploadCount());
+            imagesUploadedProgressbar.setProgressTextFormatPattern
+                    (uploadCount +"/" + levelInfo.getMaxUploadCount() );
+        }
+
+    }
+
+    private void setZeroAchievements() {
+        AlertDialog.Builder builder=new AlertDialog.Builder(this)
+                .setMessage("You haven't made any contributions yet")
+                .setPositiveButton("Ok", (dialog, which) -> {
+                    //dismiss
+                });
+        AlertDialog dialog = builder.create();
+        dialog.show();
+        imagesUploadedProgressbar.setVisibility(View.INVISIBLE);
+        imageRevertsProgressbar.setVisibility(View.INVISIBLE);
+        imagesUsedByWikiProgessbar.setVisibility(View.INVISIBLE);
+        //levelNumber.setText("LEVEL 0");
+        imageView.setVisibility(View.INVISIBLE);
+        imageByWikiText.setText("No images used");
+        imageRevertedText.setText("No image reverted");
+        imageUploadedText.setText("No images Uploaded");
+        /*BitmapDrawable bitmapImage = BitmapUtils.writeOnDrawable(bitmap, Integer.toString(levelInfo.getLevelNumber()),this);
```

Comment:
All strings should be in strings.xml file

Reply:
Fixed int the last commit